cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -31,6 +31,9 @@ spec:
   containers:
   - name: kube-proxy
     image: {{pillar['kube_docker_registry']}}/kube-proxy:{{pillar['kube-proxy_docker_tag']}}
+    resources:
+      requests:
```

Comment:
`requests` are OK for now I guess.
Depending on where we go with the QoS policies, this might make kube-proxy
have less priority. But given that it is a DaemonSet, I guess the policies
will be thwarted anyways.

On Thu, Feb 25, 2016 at 3:25 PM, Prashanth B <notifications@github.com>
wrote:

> In cluster/saltbase/salt/kube-proxy/kube-proxy.manifest
> <https://github.com/kubernetes/kubernetes/pull/22022#discussion_r54182925>
> :
>
> > @@ -31,6 +31,9 @@ spec:
> >    containers:
> >    - name: kube-proxy
> >      image: {{pillar['kube_docker_registry']}}/kube-proxy:{{pillar['kube-proxy_docker_tag']}}
> > +    resources:
> > +      requests:
>
> I don't want to curtail kube-proxy, I want it to get what it needs (plus
> I'm not 100% sure how much it will need).
>
> But I guess it doesn't make sense giving unlimited cpu to the proxy if
> nothing can run on the node, and if it bumps into the limit it's because
> something else with higher priority came along, and it'll just be slow,
> unlike memory.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/22022/files#r54182925>.
>